crates/turborepo-lib/src/run/mod.rs

```diff
@@ -76,34 +80,11 @@ impl Run {
             .validate()
             .context("Invalid package dependency graph")?;
 
-        let g = CompleteGraph::new(&pkg_dep_graph, &self.base.repo_root);
-
-        let is_single_package = opts.run_opts.single_package;
-        let turbo_json = g.get_turbo_config_from_workspace(ROOT_PKG_NAME, is_single_package)?;
-
-        opts.cache_opts.remote_cache_opts = turbo_json.remote_cache_opts.clone();
-
-        if opts.run_opts.experimental_space_id.is_none() {
-            opts.run_opts.experimental_space_id = turbo_json.space_id.clone();
-        }
-
-        let pipeline = &turbo_json.pipeline;
-
         let scm = SCM::new(&self.base.repo_root);
 
-        let mut filtered_pkgs =
+        let _filtered_pkgs =
             scope::resolve_packages(&opts.scope_opts, &self.base, &pkg_dep_graph, &scm)?;
 
-        if filtered_pkgs.len() == pkg_dep_graph.len() {
```

Comment:
Deleted because it's causing some lifetime issues and I figure this code will be handled by the actual filter/scope logic.

Reply:
Maybe leave this commented or a TODO to make sure it doesn't get forgotten since in Go land this logic is in `run` and not the filter/scope module.